qemu/tests/remote_block_resize.py

```diff
@@ -51,8 +51,13 @@ def verify_disk_size(session, os_type, disk):
     img = qemu_storage.QemuImg(data_image_params, data_dir.get_data_dir(), data_image)
     filters = {}
     data_image_dev = ""
+    img_format = data_image_params.get("image_format", "qcow2")
     if vm.check_capability(Flags.BLOCKDEV):
-        filters = {"driver": data_image_params.get("image_format", "qcow2")}
+        filters = (
+            {"image": f"{data_image}.raw"}
```

Comment:
If we run the case with qemu version<9.1.0, the resize node-name should be on the driver=raw one, not the "image: f{data_image}.raw" one.

The case is for ceph and gluster_direct only, not sure if the {"image": f"{data_image}.raw"} can adapt to both.

Reply:
Good catch. updated it to be compatible to qemu version<9.1.0 as well.
While regarding gluster_direct, I think it should be, but as gluster_direct is deprecated now. So I will not give it a try.